### PR TITLE
Codeclimate configuration + bundle update

### PR DESCRIPTION
**Pull request by steved**
* bundle update takes care of necessary Rails security upgrades
* codeclimate config ignores false positives

/cc @zendesk/samson

### CODE REVIEW

**Comment by grosser** [comment]
:+1:

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+engines:
+  brakeman:
+    enabled: true
+    exclude_fingerprints:
```
Comment (andreionut) [inline .codeclimate.yml]: how did you configured these? I don't see them mentioned in the docs (https://docs.codeclimate.com/v1.0/docs/brakeman)

**Comment by sandlerr** [comment]
rails-assets-angular-ui-bootstrap-bower, puma and angular-rails-templates all got major version bumps, I'll take a look at what changed there

File: .codeclimate.yml (hunk 1 above)
Comment (steved) [inline .codeclimate.yml]: they're mentioned in the internal security tickets, I actually haven't read through the docs

**Comment by steved** [comment]
puma one looks fine, it was just the addition of plugins

**Comment by sandlerr** [comment]
plenty of breaking changes listed at https://github.com/angular-ui/bootstrap/blob/master/CHANGELOG.md

angular-rails-templates just dropped support for Rails < 4.2

**Comment by steved** [comment]
Locked angular-ui-bootstrap to ~> 0.14, someone else can handle the upgrade.

**Pull request merged**